examples/DA_paper/1d_fes/fes.py

```diff
@@ -0,0 +1,68 @@
+import sys
+import mlptrain as mlt
+import numpy as np
+from mlptrain.box import Box
+from mlptrain.log import logger
+
+mlt.Config.n_cores = 10
+
+if __name__ == '__main__':
+
+    us = mlt.UmbrellaSampling(zeta_func=mlt.AverageDistance((1,12), (6,11)),
+                              kappa=10)
+    temp = float(300)
+
+    neb = mlt.ConfigurationSet()
+    neb.load_xyz(filename='neb_optimised.xyz', charge=0, mult=1)
```

Comment:
Such file is necessary to run US.

Reply:
It's about 220 KB, if it can be made available upon request that might be better, but if it is absolutely necessarily to be included then it might be ok - @t-young31 what are your thoughts?